WordPress/Classes/ViewRelated/Blog/BlogDetailsViewController.m

```diff
@@ -316,6 +359,13 @@ - (void)showDetailViewForSubsection:(BlogDetailsSubsection)section
     NSIndexPath *indexPath = [self indexPathForSubsection:section];
 
     switch (section) {
+        case BlogDetailsSubsectionDomainCredit:
+            self.restorableSelectedIndexPath = indexPath;
+            [self.tableView selectRowAtIndexPath:indexPath
+                                        animated:NO
+                                  scrollPosition:[self optimumScrollPositionForIndexPath:indexPath]];
+            [self showDomainCreditRedemption];
```

Comment:
I think `[self showDomainCreditRedemption]` here is not necessary and you can remove it

Reply:
I think it will be necessary once that method is implemented?

Reply:
Yep when `showDomainCreditRedemption` is implemented, the domain registration screen will be shown when this case is reached, like other cases

Reply:
This method is called by the 'WPTabBarController` and by me coordinator and, I think, is to set the right section on a split screen from another app screen (which shouldn't be necessary with this new feature and with Quick Start because both are modally presented).
The method to execute an action when you tap a row will be triggered by its `BlogDetailsRow` block (line 1103 `row.callback();`)

Reply:
Ah I see what you meant now, because of modal presentation. will update!

Reply:
👍